dev/release/release-tag.sh

```diff
@@ -92,9 +101,14 @@ echo "Creating tag $RELEASE_TAG at the head of $GIT_BRANCH"
 git tag $RELEASE_TAG
 
 # Create next version
-sed -i".tmp4" 's/version = .*$/version = '"$NEXT_VERSION"'/g' gradle.properties
-sed -i".tmp5" 's/    version=.*$/    version="'"$PYGRAVITINO_NEXT_VERSION"'",/g' clients/client-python/setup.py
-sed -i".tmp6" 's/^version = .*$/version = \"'"$NEXT_VERSION"'\"/g' clients/filesystem-fuse/Cargo.toml
+sed -i".tmp6" 's/version = .*$/version = '"$NEXT_VERSION"'/g' gradle.properties
```

Comment:
The pattern `-i".tmp"`(no space between `-i` and `".tmp6"` is only applicable on macOS and will failed in Linux.

Reply:
It should be ok for Linux. I succeed to run this on AWS Ubuntu EC2.

Reply:
One EC2 instance
```
ubuntu@ip-10-0-4-157:~$ sed -i"s/hello/nice/g" 1.txt
sed: -e expression #1, char 2: unknown command: `.'
ubuntu@ip-10-0-4-157:~$ sed -i "s/hello/nice/g" 1.txt
ubuntu@ip-10-0-4-157:~$ uname -a
Linux ip-10-0-4-157 6.8.0-1024-aws #26~22.04.1-Ubuntu SMP Wed Feb 19 06:54:57 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux
```

On my local VM
```
yuqi@ubuntu-intel:/tmp$ sed -i"s/good/hello/g" 1.txt
sed: -e expression #1, char 2: unknown command: `.'
yuqi@ubuntu-intel:/tmp$ sed -i "s/good/hello/g" 1.txt
yuqi@ubuntu-intel:/tmp$ uname -r
6.12.15-orbstack-00304-gd0ddcf70447d
```

Reply:
You use different sytax. You should add a tmp tile after `-i`.

Reply:
I see, it's my fault, and adding a temporary file works on both macOS and Linux.